### PR TITLE
Cloud rebalancer UI

### LINKED ISSUE #492: Shard rebalancing UI on cloud
## Why are we implementing it? (sales eng)

Cloud has added a GUI for shard rebalancing. Rebalancing is intentionally not done automatically when a cluster is scaled out, and a cluster administrator must run it. The graphical interface makes the process easier and provides feedback about the rebalancer progress.

### What are the typical use cases?

After adding new machines to the cluster, an admin decides to rebalance to utilize them.

### Good locations for content in docs structure

https://docs.citusdata.com/en/v7.0/cloud/performance.html#scaling-out-adding-new-nodes currently talks about the sql commands to rebalance, but we can expand it with screenshots of the UI.

**Update**: Craig suggests an "Additional Features" section in the Cloud docs, and we can put this there. Maybe we should include a link from the adding new nodes section though.

### PR DESCRIPTION
**Pull request by begriffs**
Fixes #492

### CODE REVIEW

**Comment by begriffs** [comment]
TODO: rebase this on https://github.com/citusdata/citus_docs/pull/547 when that gets merged so that the new content goes in the right section.

File: cloud/performance.rst [hunk 1]
```diff
@@ -46,20 +46,22 @@ Scaling Out (adding new nodes)
 
 Node addition completes in five to ten minutes, which is faster than node resizing because the new nodes are created without data. To take advantage of the new nodes you still must adjust manually rebalance the shards, meaning move some shards from existing nodes to the new ones.
 
-Citus does not automatically rebalance on node creation because shard rebalancing takes locks on rows whose shards are being moved, degrading write performance for other database clients. The slowdown isn't terribly severe because Citus moves data a shard (or a group of colocated shards) at a time while inserts to other shards can continue normally. However, for maximum control, the choice of when to run the shard rebalancer is left to the database administrator.
+You can go to the "Rebalancer" tab in the Cloud console to see the shard balance across nodes. This page will show a "Rebalance recommended" message if the shards could be placed more evenly, such as after a new node has been added to the cluster. Otherwise it says, "No rebalance required."
```
Comment (sumedhpathak) [inline cloud/performance.rst]: Can we add an image which shows what this message looks like? (If it is more instructive).

File: cloud/performance.rst [hunk 2]
```diff
@@ -46,20 +46,22 @@ Scaling Out (adding new nodes)
 
 Node addition completes in five to ten minutes, which is faster than node resizing because the new nodes are created without data. To take advantage of the new nodes you still must adjust manually rebalance the shards, meaning move some shards from existing nodes to the new ones.
 
-Citus does not automatically rebalance on node creation because shard rebalancing takes locks on rows whose shards are being moved, degrading write performance for other database clients. The slowdown isn't terribly severe because Citus moves data a shard (or a group of colocated shards) at a time while inserts to other shards can continue normally. However, for maximum control, the choice of when to run the shard rebalancer is left to the database administrator.
```
Comment (sumedhpathak) [inline cloud/performance.rst]: Can we also add a link to our main rebalancer section: https://docs.citusdata.com/en/v7.1/admin_guide/cluster_management.html#rebalancing-shards-without-downtime, so people can learn more about this?

Comment (begriffs) [reply cloud/performance.rst]: (Added a link, just in a slightly different place)

**Comment by begriffs** [comment]
Waiting for final review and merge of Cloud TOC reorganization, then I'll rebase and merge this one.

**Pull request merged**